### PR TITLE
ticket5 finished

### PR DESCRIPTION
**Pull request by margaritisarg**

### CODE REVIEW

File: src/components/ReviewsContainer/ButtonComments.jsx [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
+        navigate(`/review/${reviewID}`)
+    }
 
-const ButtonComments = () => {
     return(
         <div>
-            <button>Comments</button>
+            <button onClick={singleReviewPage}>Comments</button>
         </div>
     )
```
Comment (HeatherEver) [inline src/components/ReviewsContainer/ButtonComments.jsx:9]: i don't think you should be using `navigate` here or `onClick` 
we have access to `Routes` and `Link` so why not use them? 
also having to click the `comments` button to view the individual review is not very user friendly and it took me a while to figure out they were even clickable.

File: src/components/SingleReviewContainer/SingleReviewContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from "react"
+import { useParams } from "react-router"
+
+import Review from "../GenericComponents/Review"
+
+import * as api from '../../api'
+
+const SingleReviewContainer = () => {
+    
+    const [reviews, setReviews] = useState()
```
Comment (HeatherEver) [inline src/components/SingleReviewContainer/SingleReviewContainer.jsx]: this is a single review, so for readability i would change this to show that

File: src/components/SingleReviewContainer/SingleReviewContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            setIsLoading(false)
+        })
+    }, [])
+
+    return(
+        <>
+            {isLoading 
+            ? <p>Loading</p> 
+            : reviews.map(review => <Review key={review.review_id} review={review} />)}
+        </>
```
Comment (HeatherEver) [inline src/components/SingleReviewContainer/SingleReviewContainer.jsx:26]: also you're mapping over what is one item here and rendering the `Review` component means that in your single review you're not displaying the review body

**Review by HeatherEver — CHANGES_REQUESTED** [review]
Good work so far in getting the single review to render but there's a couple of points for you to look at here: 
- think about whether these `containers` are actually helping here, as you're rendering the `Review` component in your `SingleReviewContainer` but that is not rendering the reviews body to the page
- clicking the comments button to view an individual article isn't user friendly and doesn't make much sense to the user 
- make use of all `react-router-dom` has to offer, use `Links` and `Routes` then you wont have to use onClicks which saves you coding time 
- be aware of your folders and files, keeping them semantic and tidy to make it clear what each one represents 
Please can you have a look at these changes and then when refactored to use `Link` instead of `onClick` and to make sure the review body is displayed please submit another PR 😊

File: src/components/SingleReviewContainer/SingleReviewContainer.jsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            setReview(review)
+            setIsLoading(false)
+        })
+    }, [])
+
+    return(
+        <>
+            {isLoading 
+            ? <p>Loading</p> 
+            : review.map(review => <Review key={review.review_id} review={review} />)}
```
Comment (Hard-Luck) [inline src/components/SingleReviewContainer/SingleReviewContainer.jsx]: this map is still unnecessary as there is only one review in the array

**Review by Hard-Luck — COMMENTED** [review]
I would change the uneccesary map, and I noticed there is still a "more" button on the individual article, once these 2 things are cleaned up you can merge

**Pull request merged**